AzureCalling/Views/ParticipantView.swift

```diff
@@ -22,6 +22,7 @@ class ParticipantView: UIView {
     @IBOutlet weak var placeholderImage: UIImageView!
     @IBOutlet weak var videoViewContainer: UIView!
     @IBOutlet weak var participantLabel: ParticipantLabel!
+    @IBOutlet weak var participantMuteIndicator: UIView!
```

Comment:
Let's replace participantLabel with participantLabelView. And remove participantMuteIndicator.
The `participantMuteIndicator` is a subview inside `ParticipantLabelView`. We should link it in `ParticipantLabelView` class. And open a `api` in `ParticipantLabelView` to `updateMuteIndicator`

Reply:
Good suggestion, this will have separation of concerns